kolibri/core/assets/src/views/icons/KLabeledIcon.vue

```diff
@@ -1,22 +1,33 @@
 <template>
 
   <span class="labeled-icon-wrapper">
-    <div class="icon">
+    <div ref="icon" class="icon">
       <slot name="icon"></slot>
     </div>
     <div class="label">
       <slot></slot>
     </div>
+    <KTooltip
+      v-if="tooltipText"
+      reference="icon"
+      :refs="$refs"
+    >
+      {{ tooltipText }}
+    </KTooltip>
```

Comment:
new tooltip prop is now unused

let's remove this until we have a concrete need for it. Then we can decide whether it's appropriate to have the tooltip on just the icon versus on the entire labeled icon